sklearn/feature_selection/_rfe.py

```diff
@@ -296,6 +299,7 @@ def _fit(self, X, y, step_score=None, **fit_params):
         ranking_ = np.ones(n_features, dtype=int)
 
         if step_score:
+            self.n_features_selected_ = []
```

Comment:
I find this name a bit misleading: by reading just the name I would expect a final integer at the end of the RFE process.

Maybe `n_features_at_step_` or `n_features_per_iter_` would be more intuitive?

Reply:
Or we could rename both attributes (they are private, right?) to something like: `step_scores_` and `step_n_features_`.

Reply:
Like that better as well. Just pushed those changes in the latest commit.